insalan/tournament/models/seat.py

```diff
@@ -14,6 +14,13 @@ class Seating(models.Model):
         on_delete=models.CASCADE,
         verbose_name=_("Évènement"),
     )
+    slot = models.ForeignKey(
```

Comment:
You should not link a seat to a slot but the opposite with a list of Seats in the Slot model.
To understand which object should be referencing which one , you need to think about the connection between them. The Slot will be the object sent to the front and their respective seats needs to be available (to display it onto the front page). If you reverse this schema, you'll have to send a list of seat with their relatives info which will duplicate a lots of things and is not really good.
You can use a ManyToMany field in SeatSlot to do it.